### PR TITLE
feat: adding the ability to include a custom header on generated files

### PR DESCRIPTION
**Pull request by kaolin**
Feature request, I'm not wed to this particular implementation but figured it was the easiest way to ask. ^_^

### CODE REVIEW

**Review by sibelius — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -76,6 +76,20 @@ Note that this does mean you need to configure Babel to transform the ES module
...
+#### Custom Headers
+
+If you need to add a custom header to generated files, perhaps for a custom linter
+or to get boilerplate license code in, that can be passed in also in compilerOptions 
+as `customHeader`:
+
+```json
+{
+  "compilerOptions": {
+    "customHeader": "/* &copy; 2021 Example.org - @generated code */"
```
Comment (maraisr) [inline README.md:88]: the relay compiler calls this the `banner` — perhaps we call it this as well, instead of customHeader

**Review by maraisr — CHANGES_REQUESTED** [review]
Thanks for the PR mate, looks good! Just one thing I think we should change.

File: README.md (hunk 1 above)
Comment (kaolin) [reply README.md:88]: Sounds great! :) update shortly unless I get too distracted by dinner happening around me ;)

**Comment by kaolin** [comment]
bump? :)

**Review by saihaj — APPROVED** [review]
Thanks @kaolin can you please rebase this?

**Comment by kaolin** [comment]
I got tripped up on rebase conflicts (and git segfaulting and vim locking up!?), so squashed the changes for simplicity. :)

**Pull request merged**